database.ini

```diff
@@ -1,12 +1,12 @@
+;[postgresql]
+;host = localhost
+;database = food_db
+;user = postgres
+;password = admin
+
 [postgresql]
 host = localhost
 database = food_db
 user = postgres
 password = admin
```

Comment:
it's a good practice to hide sensitive data as DB credentials into .env and then use load_dotenv to get the credentials. Right now you have your DB on localhost, but once you publish it on the server it may cause significant security issues.
I am not sure that food_db is a good name, because your DB also contains information about users. So I suggest renaming it to something like "fitness_calculator".